android/java/org/chromium/base/BraveFeatureList.java

```diff
@@ -31,6 +31,6 @@ public abstract class BraveFeatureList {
     public static final String AI_CHAT = "AIChat";
     public static final String BRAVE_SHOW_STRICT_FINGERPRINTING_MODE =
             "BraveShowStrictFingerprintingMode";
-    public static final String BRAVE_ZERO_DAY_FLAG_ANDROID = "BraveZeroDayFlagAndroid";
+    public static final String BRAVE_DAY_ZERO_EXPERIMENT = "BraveDayZeroExperiment";
```

Comment:
@DJAndries the `BRAVE_DAY_ZERO_EXPERIMENT`variable is the same variable cross platforms and as a replacement for the previous Android-specific one?

Reply:
yes, correct